sdk/core/Azure.Core/src/Cryptography/IKeyEncryptionKey.cs

```diff
@@ -12,6 +12,11 @@ namespace Azure.Core.Cryptography
     /// </summary>
     public interface IKeyEncryptionKey
     {
+        /// <summary>
+        /// The Id of the key used to perform cryptographic operations for the client.
+        /// </summary>
+        Uri KeyId { get; }
```

Comment:
Given the changes I'm making now, should this be just `Id`?

Reply:
I think I'd like to leave this as KeyId, because this is an interface and depending on the name of the class implementing this interface, it might not be entirely obvious what Id is referring to.  For instance CryptographyClient implements this interface, but if we had a property CryptographyClient.Id it wouldn't be obvious that this is referring to the key used to by the CryptographyClient rather than some Id of the CryptographyClient itself.